content/release-10-16-0/platform-10-16-0-bundle/10-16-0-0.md

```diff
@@ -63,7 +71,32 @@ MTM-46640</td>
 
 <tr>
 <td>
- MongoDB </td>
+Core platform</td>
+<td>The identifier for managed objects is now a random number and no longer monotonically increasing.</td>
+<td>
+MTM-47763</td>
+</tr>
+
+<tr>
+<td>
+Data Broker</td>
+<td>The databroker-agent server microservice (a key part of the microservice-based data broker) has been upgraded to use the more secure Cumulocity IoT microservice API version 2.
+Additionally, a small change has been made to improve an existing error message logged when forwarding to the destination tenant fails.</td>
+<td>
+MTM-47731</td>
+</tr>
+
+<tr>
+<td>
+Inventory roles</td>
+<td>For users with inventory role access only, the performance for GET requests for operations has been improved. The feature is hidden behind a feature toggle which is already used for other improvements. The tenant option to enable/disable the toggle has the following payload: "category": "configuration", "key": "acl.algorithm-version", "value": "OPTIMIZED"/"LEGACY". For details, see the <a href="https://www.cumulocity.com/api" class="no-ajaxy">Cumulocity IoT OpenAPI Specification</a>.</td>
```

Comment:
is the payload JSON (looks like it)?
is it the full payload? could it be rendered as a code block? if it's the full payload, then i think "{ }" wrapping it could be added.